boilingcore/text_helpers.go

```diff
@@ -19,6 +19,7 @@ type TxtToOne struct {
 	}
 
 	ForeignTable struct {
+		Name         string
```

Comment:
I don't actually see anywhere where this is used. Also this data is available already on the ForeignKey and Relationship structs so I'm not sure this is necessary?